apidoc/Titanium/UI/Attribute.yml

```diff
@@ -115,3 +124,119 @@ properties:
+platforms: [iphone, ipad]
+since: "7.3.0"
+  
+properties:
+  - name: alignment
+    summary: The text alignment of the receiver.
+    type: Number
+    constants: [ Titanium.UI.TEXT_ALIGNMENT_* ]
+
+  - name: firstLineHeadIndent
+    summary: The indentation of the first line of the receiver.
+    description: |
+      This property contains the distance (in points) from the leading margin 
+      of a text container to the beginning of the first line of paragraph. 
+      This value is always nonnegative float value.
+    type: Number
+
+  - name: headIndent
+    summary: The indentation of the lines of lines other than the first.
+    description: |
+      This property contains the distance (in points) from the leading margin 
+      of a text container to the beginning of lines other than the first. 
+      This value is always nonnegative float value.
+    type: Number
+
+  - name: tailIndent
+    summary: The trailing indentation of the receiver.
+    description: |
+      If positive, this value is the distance from the leading margin 
+      (for example, the left margin in left-to-right text). If 0 or negative, 
+      it is the distance from the trailing margin. This is a float value.
+    type: Number
+
+  - name: lineBreakMode
+    summary: The mode that should be used to break lines in the receiver.
+    description: |
+      This property contains the line break mode to be used laying out the text of paragraph.
+    type: Number
+    constants: [ Titanium.UI.ATTRIBUTE_LINE_BREAK_* ]
+
+  - name: maximumLineHeight
+    summary: The maximum line height of receiver.
+    description: |
+      This property contains the maximum height in points that any line in the receiver will occupy, 
+      regardless of the font size or size of any attached graphic. This value is always nonnegative. 
+      The default value is 0.
+    type: Number
+
+  - name: minimumLineHeight
+    summary: The minimum height of receiver.
+    description: |
+      This property contains the minimum height in points that any line in the receiver will occupy, 
+      regardless of the font size or size of any attached graphic. This value must be nonnegative.
+    type: Number
+
+  - name: lineSpacing
+    summary: The distance in points between the bottom of one line fragment and the top of the next.
+    description: |
+      This value is always nonnegative. This value is included in the line fragment heights 
+      in the layout manager.
+    type: Number
+
+  - name: paragraphSpacing
+    summary: The space after the end of the paragraph.
+    description: |
+      This property contains the space (measured in points) added at the end of the paragraph 
+      to separate it from the following paragraph. This value must be nonnegative. The space 
+      between paragraphs is determined by adding the paragraphSpacing of previous paragraph and 
+      the paragraphSpacingBefore of current paragraph.
+    type: Number
```

Comment:
Perhaps we should rename property `paragraphSpacing` to `paragraphSpacingAfter`. This makes it clear that it's only applied after the paragraph (assuming this is actually how it works). And it clearly separates it from the `paragraphSpacingBefore` property.